frontend/src/components/HospitalAdminDetail.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import AppointmentScheduler from './AppointmentScheduler'
+
+export default () => {
+
+    const codeForm = {
+        
+    }
+
+  return (
+    <div style={{display: 'flex'}}>
+      <div style={{textAlign: 'center', width: '50%'}}>
+        <input type="text" placeholder="Insert six digit code"/>
+        <br/>
+        <button>Join Visit</button>
+        <br/>
+        <button>Upload results</button>
+      </div>
+      <div>
+        <AppointmentScheduler />
```

Comment:
You should use `react-calendar` directly rather than `AppointmentScheduler`. This calendar is not an appointment scheduler, and the `AppointmentScheduler` is much more than a calendar; it's parametrized by the URL parameters and serves as a route-level component.